hubspot/plugin.py

```diff
@@ -1054,46 +1376,211 @@ def hubspot_update_companies(
 
 Returns:
 Company IDs that have been updated.
 
-The following are default properties (and types) in Hubspot:
+The property names to update MUST be one of the following:
 
 Default Properties of Type String:
-{", ".join(_COMPANY_PROPERTIES_STRING)}
+about_us
+first_conversion_event_name
+founded_year
+hs_additional_domains
+hs_all_assigned_business_unit_ids
+hs_analytics_first_touch_converting_campaign
+hs_analytics_last_touch_converting_campaign
+hs_analytics_latest_source
+hs_analytics_latest_source_data_1
+hs_analytics_latest_source_data_2
+hs_analytics_source
+hs_analytics_source_data_1
+hs_analytics_source_data_2
+hs_annual_revenue_currency_code
+hs_avatar_filemanager_key
+hs_ideal_customer_profile
+hs_last_sales_activity_type
+hs_merged_object_ids
+hs_notes_next_activity_type
+hs_object_source
+hs_object_source_detail_1
+hs_object_source_detail_2
+hs_object_source_detail_3
+hs_object_source_id
+hs_object_source_label
+hs_pipeline
+hs_target_account
+hs_target_account_recommendation_state
+hs_unique_creation_key
+hs_user_ids_of_all_notification_followers
+hs_user_ids_of_all_notification_unfollowers
+hs_user_ids_of_all_owners
+recent_conversion_event_name
+timezone
+total_money_raised
+name
+owneremail
+twitterhandle
+ownername
+phone
+twitterbio
+address
+address2
+facebook_company_page
+city
+linkedin_company_page
+linkedinbio
+state
+googleplus_page
+engagements_last_meeting_booked_campaign
+engagements_last_meeting_booked_medium
+engagements_last_meeting_booked_source
+hubspot_owner_id
+zip
+country
+hubspot_team_id
+hs_all_owner_ids
+website
+domain
+hs_all_team_ids
+hs_all_accessible_team_ids
+industry
+lifecyclestage
+hs_lead_status
+type
+description
+web_technologies
 
 Default Properties of Type Number:
-{", ".join(_COMPANY_PROPERTIES_NUMBER)}
+facebookfans
+hs_analytics_num_page_views
+hs_analytics_num_page_views_cardinality_sum_e46e85b0
+hs_analytics_num_visits
+hs_analytics_num_visits_cardinality_sum_53d952a6
+hs_created_by_user_id
+hs_customer_success_ticket_sentiment
+hs_num_blockers
+hs_num_contacts_with_buying_roles
+hs_num_decision_makers
+hs_num_open_deals
+hs_object_id
+hs_object_source_user_id
+hs_pinned_engagement_id
+hs_predictivecontactscore_v2
+hs_predictivecontactscore_v2_next_max_max_d4e58c1e
+hs_source_object_id
+hs_target_account_probability
+hs_time_in_customer
+hs_time_in_evangelist
+hs_time_in_lead
+hs_time_in_marketingqualifiedlead
+hs_time_in_opportunity
+hs_time_in_other
+hs_time_in_salesqualifiedlead
+hs_time_in_subscriber
+hs_total_deal_value
+hs_updated_by_user_id
+num_associated_contacts
+num_associated_deals
+num_conversion_events
+num_conversion_events_cardinality_sum_d095f14b
+recent_deal_amount
+total_revenue
+twitterfollowers
+num_contacted_notes
+num_notes
+numberofemployees
+annualrevenue
+hs_parent_company_id
+hs_num_child_companies
+hubspotscore
+days_to_close
 
 Default Properties of Type Datetime:
-{", ".join(_COMPANY_PROPERTIES_DATETIME)}
+closedate_timestamp_earliest_value_a2a17e6e
```

Comment:
these values with a2a17e6e and other hashes feel like they might be generated for an account specifically